packages/aws-cdk-lib/aws-autoscaling/lib/step-scaling-policy.ts

```diff
@@ -112,7 +112,7 @@ export class StepScalingPolicy extends Construct {
       const threshold = intervals[alarms.lowerAlarmIntervalIndex].upper;
 
       this.lowerAction = new StepScalingAction(this, 'LowerPolicy', {
-        adjustmentType: props.adjustmentType,
+        adjustmentType,
```

Comment:
The property is defined [here](https://github.com/aws/aws-cdk/blob/v2.122.0/packages/aws-cdk-lib/aws-autoscaling/lib/step-scaling-policy.ts#L105).